src/qt_gui/game_list_frame.h

```diff
@@ -102,10 +119,7 @@ public Q_SLOTS:
     static bool CompareStringsDescending(GameInfo a, GameInfo b, int columnIndex) {
         switch (columnIndex) {
         case 1: {
-            std::string name_a = a.name, name_b = b.name;
-            std::transform(name_a.begin(), name_a.end(), name_a.begin(), ::tolower);
-            std::transform(name_b.begin(), name_b.end(), name_b.begin(), ::tolower);
-            return name_a > name_b;
+            return GameListFrameHelper::CompareStringsCaseInsensitive(a.name, b.name);
```

Comment:
The order is reverse here compared to other place. Probably need to swap the a and b argument order